CONTRIBUTING.md

```diff
@@ -33,7 +33,15 @@ Follow these steps to build a new version of the plugin:
 
 ### Environment setup
 
-Make sure you can build the plugin by following the steps above.
+First, make sure you can build the plugin by following the steps above.
+
+Then, make sure you have installed ```Gulp```, ```Mocha```, and ```Mocha JUnit Reporter``` through npm.
+
+```
+npm install -g gulp
+npm install -g mocha
+npm install -g mocha-junit-reporter
```

Comment:
Can we add these to `devDependencies` and run them from the `bin` location? Gulp is probably required to be global, but the others most likely don't. For example, for a local `mocha` install, you can run it from `node_modules/mocha/bin/mocha` without having to install it globally. This way, developers do `npm install` and they are done with the setup, since that installs devDependencies directly.